docs/book/getting_started/installation_and_setup.md

```diff
@@ -1,16 +1,81 @@
 # Installation and Setup
 
+Kubebuilder requires multiple binaries to be installed and cannot be installed with `go get`.
+
 {% method %}
 
+## Installing a stable release
+
 Install kubebuilder by downloading the latest stable release from the
-[github repo](https://github.com/kubernetes-sigs/kubebuilder/releases)
-or by running `go get` to get the latest code.
+[github repo](https://github.com/kubernetes-sigs/kubebuilder/releases).
+
+{% sample lang="mac" %}
+```bash
+version=1.0.0 # latest stable version
```

Comment:
Not a blocker.
I suggest having a place to hold the latest stable version like https://storage.googleapis.com/kubernetes-release/release/stable.txt
So we don't hardcode the version in the doc. Instead, use curl to get the latest version.

Reply:
installation instructions in current form are not the best experience. We will have to replace these with a `install.sh` script which users `curl <script-path> | sh`.